src/main/kotlin/org/jitsi/jibri/util/Tee.kt

```diff
@@ -59,9 +69,10 @@ class TeeLogic(inputStream: InputStream) {
      * starting from the time of its creation
      */
     fun addBranch(): InputStream {
-        val outputStream = PipedOutputStream()
-        branches.add(outputStream)
-        return PipedInputStream(outputStream)
+        PipedOutputStream().let {
```

Comment:
Just curious: are these two equivalent? Is the purpose of using "let" here just to scope the PipedOutputStream instance?

Reply:
yeah, it just saves declaring the variable basically.

actually i wonder if this would be better:
```
with(PipedOutputStream()) {
    branches.add(this)
    return PipedInputStream(this)
}
```